exec/java-exec/src/main/java/org/apache/drill/exec/store/easy/json/JSONRecordReader.java

```diff
@@ -189,39 +191,33 @@ private long currentRecordNumberInFile() {
   public int next() {
     writer.allocate();
     writer.reset();
-
     recordCount = 0;
     ReadState write = null;
-//    Stopwatch p = new Stopwatch().start();
-    try{
-      outside: while(recordCount < DEFAULT_ROWS_PER_BATCH) {
-        writer.setPosition(recordCount);
-        write = jsonReader.write(writer);
-
-        if(write == ReadState.WRITE_SUCCEED) {
-//          logger.debug("Wrote record.");
-          recordCount++;
-        }else{
-//          logger.debug("Exiting.");
-          break outside;
-        }
-
+    outside: while(recordCount < DEFAULT_ROWS_PER_BATCH){
+    try
+      {
+            writer.setPosition(recordCount);
```

Comment:
seems this is still doing indent of 4.  We use 2 spaces (see https://drill.apache.org/docs/apache-drill-contribution-guidelines/   scroll down to Step 2).   Did it pass the mvn command line build without checkstyle violations ?